src/__snapshots__/index.test.js.snap

```diff
@@ -10,7 +10,10 @@ Object {
   "PopupDefaultInsert": [Function],
   "VectorLayer": [Function],
   "ZoomControls": [Function],
+  "centerAndZoom": [Function],
   "connectToMap": [Function],
+  "createMap": [Function],
+  "createSelectInteraction": [Function],
```

Comment:
these were the three missing functions that are now accurately added to the root export snapshot